pick.c

```diff
@@ -814,6 +827,10 @@ get_key(char *buf, size_t size, size_t *nread)
 		KEY(LEFT,	"\033[D"),
 		KEY(PAGE_DOWN,	"\033[6~"),
 		KEY(PAGE_UP,	"\033[5~"),
+		KEY(HOME,	"\033[7~"),
```

Comment:
"rxvt"-terminals use "\033[7~" & "\033[8~" for "Home"/"End", I use "rxvt-unicode" and "infocmp" shows: "khome=\E[7~", "kend=\E[8~" ...